chia/rpc/wallet_rpc_api.py

```diff
@@ -1200,9 +1199,7 @@ async def nft_transfer_nft(self, request):
         sb = await nft_wallet.transfer_nft(
             request["nft_coin_info"],
             bytes.fromhex(request["new_did"]),
-            request["new_did_parent"],
             request["new_did_inner_hash"],
```

Comment:
Should `new_did_inner_hash` and `trade_price` be optional params? Something like

`request.get("new_did_inner_hash")`
`request.get("trade_price", 0)`

Though it's not clear to me that `transfer_nft` is setup to accept None for the inner puzzlehash.

Reply:
Will address this separately